README.md

```diff
@@ -164,13 +164,25 @@ by calling `require`:
 var escomplex = require('escomplex');
 ```
 
-It exports one function,
-called `analyse`:
+It exports two functions,
+the primary interface, called `analyse`:
 
 ```javascript
 var result = escomplex.analyse(ast, walker, options);
 ```
 
+and a second function, called `processResults`:
+
+``` javascript
+escomplex.processResults(result, false);
+```
+
+This function takes a report object and computes
+aggregate scores for all individual files and also adjaceny and
+visibility matrices. The second parameter corresponds to `noCoreSize`
```

Comment:
So something I am a massive fan of in READMEs and comments is Brian Kernighan's semantic line-break policy.

It sounds really OCD/anal, but it definitely pays off down the line whenever you need to diff text changes. I read about it [here](http://rhodesmill.org/brandon/2012/one-sentence-per-line/).

I'm not so fussed about it that I try to ram it down people's throats and I know some people I've worked with think I'm weird for even thinking about text formatting, but I always like to share it with people in case they approve. :)